cogs/induct.py

```diff
@@ -183,6 +182,13 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
         # NOTE: Shortcut accessors are placed at the top of the function, so that the exceptions they raise are displayed before any further errors may be sent
         guest_role: discord.Role = await self.bot.guest_role
```

Comment:
Add `main_guild` shortcut accessor to top of function (before `guest_role`) for fail-fast execution:

```python
main_guild: discord.Guild = self.bot.main_guild
```